aeron-driver/src/main/java/io/aeron/driver/MinMulticastFlowControl.java

```diff
@@ -32,40 +27,13 @@
  * Tracking of receivers is done as long as they continue to send Status Messages. Once SMs stop, the receiver tracking
  * for that receiver will timeout after a given number of nanoseconds.
  */
-public class MinMulticastFlowControl implements FlowControl
+public class MinMulticastFlowControl extends TaggedMulticastFlowControlBase
```

Comment:
Tagged should extend Min as it added new features.

Reply:
I think the problem is with the naming of the base class.  I'm going to rework is slightly.  I see Tagged and Min as peers.  Just with different approaches to as to whether the receiver is in the group.  Min - always, Tagged - if receiverTag matches.